runner/ansible/roles/2.2.4/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '2.2.4'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    # Check the pacemaker version IS
```

Comment:
Typo in the comment `pacemaker -> corosync`

Reply:
Done